bundles/org.openhab.binding.energidataservice/README.md

```diff
@@ -579,6 +579,69 @@ var result = edsActions.calculateCheapestPeriod(time.Instant.now(), time.Instant
 
 :::
 
+::: tab JRuby
+
+```ruby
+energi = things["energidataservice:service:energidataservice"]
+
+price_map = energi.get_prices
+hour_start = ZonedDateTime.now.to_instant.truncated_to(ChronoUnit::HOURS)
+logger.info "Current total price excl. VAT: #{price_map[hour_start]}"
+
+price_map = energi.get_prices("SpotPrice,GridTariff")
+logger.info "Current spot price + grid tariff excl. VAT: #{price_map[hour_start]}"
+
+price = energi.calculate_price(Instant.now, 1.hour.from_now.to_instant, 150 | "W")
```

Comment:
why `ZonedDateTime.now.to_instant` above, but `Instant.now` her?

Reply:
Good catch, the above should be Instant.now too.